common/density/rectangular_prism.h

```diff
@@ -0,0 +1,33 @@
+/* 
+ * File:   rectangular_prism.h
+ * Author: patrick
+ *
+ * Created on February 16, 2015, 4:20 PM
+ */
+
+#ifndef RECTANGULAR_PRISM_H
+#define	RECTANGULAR_PRISM_H
+
+class RectangularPrism
```

Comment:
Eh, not really a big deal. Pretty sure we've all had a teacher mandate getters and setters on us before, so anyone looking at this should know it's just aggregate data.